src/ClientData/TimerDataTest.cpp

```diff
@@ -21,144 +21,174 @@ TEST(TimerData, IsEmpty) {
   EXPECT_EQ(timer_data.GetMinTime(), std::numeric_limits<uint64_t>::max());
 }
 
+/*
+Timers to test:
+_____________________________
+|   Left   |     |  Right   |  |
+|-----------------------------
+     |  Middle / Down   |
+     --------------------
+ */
+
+static constexpr uint64_t kLeftTimerStart = 2;
+static constexpr uint64_t kLeftTimerEnd = 6;
+static constexpr uint64_t kMiddleTimerStart = 5;
+static constexpr uint64_t kMiddleTimerEnd = 9;
+static constexpr uint64_t kRightTimerStart = 8;
+static constexpr uint64_t kRightTimerEnd = 12;
+static constexpr uint64_t kNumTimers = 3;
+static constexpr uint64_t kDepth = 2;
+static constexpr uint64_t kMinTimestamp = 2;
+static constexpr uint64_t kMaxTimestamp = 12;
+
+TimerInfo GetLeftTimer() {
```

Comment:
I have to admit, I still feel don't feel 100% confident in the `static` and `constexpr` topic. But I am going to give it a shot. 

`static` (outside of a class) means internal linkage, aka only visible in the compilation unit. `constexpr` already has internal linkage, so `static constexpr` is redundant here and I would change it to just `constexpr`. 

For the functions, I would add `static` to enforce internal linkage. (Maybe the function can even be `constexpr`, but probably not, since its not a trivial `struct` but a proto)

Reply:
Thanks for the detailed comment Anton. I'm not very consistent using static, but I think that I learned from your comment.